Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
Someone else would know better of more cases of resolution/aspect ratio changes. The only one I know of is the change right at the start of the game. By the way, the internal resolution, at least for SSBM, is 642x528 which isn't any nice ratio.